common/dex-auth/authorization/Istio/cluster_rbac_config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ClusterRbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
```

Comment:
Any service or component specified under 'inclusion' would be subject to RBAC. Since this is an example for 'ml-pipeline', I haven't kept it cluster-wide.